sidecar/tests/test_accel.py

```
@@ -1171,3 +1104,492 @@ def test_models_catalog_variant_ids(monkeypatch):
+    m = _machine(installed=frozenset())
+    import pytest as _pytest
+    with _pytest.raises(accel.NoUsablePlan):
+        accel.resolve("whisper-base", machine=m)
+
+
+# ── Phase E1: GPU identity + fresh memory reads ──────────────────────────────
+
+
+class _FakeTcDev:
+    def __init__(self, kind, desc, total, free, device_type="gpu"):
+        self.kind = kind; self.description = desc
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)